### DISCUSSION TITLE
Rendering Chart on tooltip.

### DISCUSSION THREAD

**Discussion by bhumisha** [flant/grafana-statusmap#167] [Q&A]
Hi,

This is an amazing plugin and really helping in my work. I have to embed status map iframe to that tooltip feature. and when I tried in code (customContent) , I am able to see the embedded panel. but I am not able to configure as mentioned in your comment. 

Could you please help to get idea how to load that thru my datasource.

**Comment by diafour** [comment]
Hello! Please, send a screenshot or a sketch of what you trying to achieve. Also, please, send an error message if there is one.

**Reply by bhumisha** [reply]
Hey , I am trying to achieve two things, 
1) I wanted to give borders to each element in StatusMap and also wanted to move x-axis table as top , so it give bit of tabular view. I am attaching image for reference
<img width="316" alt="BorderEachElementToStatusMap" src="https://user-images.githubusercontent.com/20823163/107584818-00ff1580-6bb2-11eb-8e52-c22315c4abae.png">

Second, I am embedding <iframe src="http://localhost:3022/d/PGM48xTMz/poup-one?orgId=1&refresh=1m&from=1612903115875&to=1612903415875&viewPanel=15 width=“600" height=“400" frameborder="0"></iframe> in tooltip customContent section. but it is not available in last release. How can I achieve this without modifying plugin.
<img width="1213" alt="tooltipcustomcontent" src="https://user-images.githubusercontent.com/20823163/107684380-90123900-6c57-11eb-8d9f-d5b7800473a6.png">
Hope this give more clarity to my query.

**Reply by diafour** [reply]
1. Borders and top x-axis require plugin modification.
2. A "Custom content" feature is not released yet, new release requires signing and a new building process. Right now you can install the plugin from the master branch.

**Reply by bhumisha** [reply]
Thanks for update.

I have one more issue, when I am embedding dashboard to tooltip. I need parameter also from that dashboard so not taking Shared Embded <iframe> but I am just taking URL for sharing. And now I am getting Dashbaord name and Back Arrow (you can see in above screen -Popup one name) How can I hide those. Kiosk=TV is not helping here. 

Thanks in Advance 
Bhumisha

**Reply by diafour** [reply]
That arrow seems like an edit mode... I've added "kiosk" to the dashboard's URL and the header is gone.

<img width="861" alt="Снимок экрана 2021-02-18 в 22 19 33" src="https://user-images.githubusercontent.com/1055474/108409777-ae6bcc00-7237-11eb-83ae-eb62f8ba2d6d.png">

**Reply by bhumisha** [reply]
Thanks for reply.  In popup,I need to show parameters for heatMap.  User can even change some of the parameter in tooltip.
with Kiosk, it is view only chart and with Kiosk=tv , it is loading actual dashboard view url map. which includes names and other parameter too. 

From navbar class div, I need to show navbar-buttons div only. the navbar-edit and name , I need to hide.

<img width="1213" alt="107684380-90123900-6c57-11eb-8d9f-d5b7800473a6" src="https://user-images.githubusercontent.com/20823163/108924884-6ad6e080-75f0-11eb-96b0-fa159acae04c.png">

**Reply by diafour** [reply]
I've found a list of available modes for a dashboard: https://grafana.com/docs/grafana/latest/dashboards/playlist/#start-a-playlist and there is no option to display controls with the hidden navbar. So, you better ask for a solution/workaround in the Grafana repository.

**Reply by bhumisha** [reply]
Thank you. I have ask for solution in Grafana repo. 
I tried Kiosk but I need timeseries parameters . Just need to hide title and left menu option.
